CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v2.0.2
+
+### 🐞 Bug fixes
+
+- Fixed issues with iOS binaries.
```

Comment:
Is that a change in MapLibre GL Native?

Reply:
Yes, but it's relevant here. See https://github.com/maplibre/maplibre-gl-native/pull/517